packages/leaflet-map/src/base-map.tsx

```diff
@@ -155,11 +153,11 @@ export function BaseMap({
 
   }, [markers]);
 
-  async function setBoundsAndCenter(points: Object) {
+  async function setBoundsAndCenter(points: LocationType[] | MarkerProps[]) {
 
 	  let poly = [];
     if (points && Array.isArray(points)) {
-	    points.forEach(function(point) {
+	    points.forEach(function(point: any) {
```

Comment:
Denk dat je beter :any  in de forEach kan vervangen met :LocationType | MarkerProps. Dan behoud je de error en lint hinting in je IDE mocht de structuur van LocationType of MarkerProps veranderen.

Reply:
Nu met een heel eigen type